### PR TITLE
Add redirect for /current path

### LINKED ISSUE #17906: Incorrect redirect in docs page
https://trino.io/docs/current redirects to https://trinodb.github.io/docs.trino.io/current/
the browser should stay in `https://trino.io/*`

### PR DESCRIPTION
**Pull request by mosabua**
Should fix https://github.com/trinodb/trino/issues/17906

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <html>
 <head>
     <meta charset="utf-8">
-    <title>Presto | Documentation</title>
```
Comment (martint) [inline index.html:5]: Unrelated to this commit. It should be in a separate commit.

Comment (mosabua) [reply index.html:5]: Ok

**Comment by mosabua** [comment]
Commits are separated now @martint

**Review by martint — APPROVED** [review]

**Comment by mosabua** [comment]
I do not have access to merge this @martint

**Comment by mosabua** [comment]
Sweet .. that worked in production.
